src/view/screens/PostThread.tsx

```diff
@@ -34,6 +34,7 @@ export const PostThreadScreen = withAuthRequired(({route}: Props) => {
       const threadCleanup = view.registerListeners()
 
       InteractionManager.runAfterInteractions(() => {
+        store.shell.setMinimalShellMode(false)
```

Comment:
Need this?

Reply:
was testing something, didnt work, will remove